explainer.md

```diff
@@ -16,10 +18,22 @@ invocation of a payment handler (third-party service) that may be either a nativ
 (e.g. ApplePay from Safari) or a worker from an origin different to the calling
 context.
 
-This necessitated the introduction of a new UI component that allows the
-worker to display UI to the user in the form of a modal window with the privileges of a top-level
-browsing context.
+This necessitated the introduction of: 
+1. a means for web applications to indicate what actions they can "handle", beyond being just a regular (installable) website.   
```

Comment:
Sounds like the old Web Intents project. Could you add that to the prior art section, please?

Reply:
We should probably avoid the use of the legal term "prior art" - but we could add something to "Existing Web Platform solutions" (or change that to legacy solutions).

Reply:
Added links below